docs/guides/getting-started.md

```diff
@@ -29,7 +29,7 @@ Here is an example of how to configure the client:
 import MedrunnerApiClient from "@medrunner/api-client";
 
 const apiConfig = {
-  token: "YOUR_REFRESH_TOKEN",
+  refreshToken: "YOUR_REFRESH_TOKEN",
```

Comment:
Not your fault, but while we're here we should probably update this to `YOUR_API_TOKEN` since that's kind of the whole point of this client lol